CreateOwnAdventure

```diff
@@ -0,0 +1,8 @@
+Hello avid travelers. You must be starved. 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.